time timestamp
float32 stateOfCharge
bool dataOk
